app/views/layouts/application.html.erb

```diff
@@ -51,9 +51,6 @@
 
     </ul>
   </div>
-  <% if request.env['PATH_INFO'] == "/users/#{@user.id}" %>
-    <p class="fas fa-pencil-alt"><%= link_to '  edit', edit_user_path(@user)%></p>
-  <% end %>
 </div>
```

Comment:
Did we change this for editing a user's profile?

Reply:
This is no longer inside the application.html.erb because it was causing issues. It now lives inside the user show page. You will notice when you merge Smitha and I's  pull request that the edit button is floating somewhere on the top right. That is a result of that change, that still needs some attention.

Reply:
Ahhhh okay thank you!